src/unittests/sr5.CharacterImporter.spec.ts

```diff
@@ -0,0 +1,187 @@
+                          "min": "1",
+                          "max": "6",
+                          "aug": "10",
+                          "bp": "10",
+                          "metatypecategory": "Standard"
+                        },
+                        {
+                          "name_english": "INT",
+                          "name": "INT",
+                          "base": "6",
+                          "total": "6",
+                          "min": "1",
+                          "max": "6",
+                          "aug": "10",
+                          "bp": "0",
+                          "metatypecategory": "Standard"
+                        },
+                        {
+                          "name_english": "LOG",
+                          "name": "LOG",
+                          "base": "4",
+                          "total": "4",
+                          "min": "1",
+                          "max": "6",
+                          "aug": "10",
+                          "bp": "0",
+                          "metatypecategory": "Standard"
+                        },
+                        {
+                          "name_english": "WIL",
+                          "name": "WIL",
+                          "base": "5",
+                          "total": "5",
+                          "min": "1",
+                          "max": "6",
+                          "aug": "10",
+                          "bp": "0",
+                          "metatypecategory": "Standard"
+                        },
+                        {
+                          "name_english": "EDG",
+                          "name": "EDG",
+                          "base": "3",
+                          "total": "3",
+                          "min": "2",
+                          "max": "7",
+                          "aug": "7",
+                          "bp": "15",
+                          "metatypecategory": "Special"
+                        },
+                        {
+                          "name_english": "MAG",
+                          "name": "MAG",
+                          "base": "7",
+                          "total": "7",
+                          "min": "6",
+                          "max": "12",
+                          "aug": "12",
+                          "bp": "35",
+                          "metatypecategory": "Special"
+                        },
+                        {
+                          "name_english": "ESS",
+                          "name": "ESS",
+                          "base": "1",
+                          "total": "1",
+                          "min": "1",
+                          "max": "6",
+                          "aug": "6",
+                          "bp": "0",
+                          "metatypecategory": "Standard"
+                        }
+                      ]
+                    }
+                  ]
+            }
+        }
+    }
```

Comment:
Anyone got any idea how to have a pseudo chummer file without adding all the text?